### PR TITLE
Finnish translation for paru

### PR DESCRIPTION
**Pull request by keinhen**
Simple translation, some technical terminology (make / devel / snapshot) was left unchanged for clarity over full translation, am open to suggestions if there are any commonly used terms for these!

Tried to follow pacman's own finnish translation for reference on other technical terminology such as repository (pakettivarasto), but in some parts such as "repo package", tried to avoid redundancy of "pakettivarasto paketti" with simpler, "varasto paketti", but am not sure if this should just be replaced by "repo paketti" or something more simple and similar.

### CODE REVIEW

File: po/fi.po [hunk 1]
```diff
@@ -0,0 +1,688 @@
...
+msgid "yes"
+msgstr "kyllä"
+
+#: src/util.rs:103
+msgid "n"
+msgstr "e"
+
+#: src/util.rs:103
+msgid "no"
+msgstr "Ei"
```
Comment (Morganamilo) [inline po/fi.po:668]: Case needs to be maintained.

File: po/fi.po [hunk 2]
```diff
@@ -0,0 +1,688 @@
...
+
+#: src/install.rs:762
+msgid "failed to read dir: {}"
+msgstr "ei pystytty lukemaan kansiota: {}"
+
+#: src/install.rs:778
+msgid "{} is a directory\n"
+"{} on kansio\n"
+""
+msgstr ""
```
Comment (Morganamilo) [inline po/fi.po:413]: Not translated.

File: po/fi.po [hunk 3]
```diff
@@ -0,0 +1,688 @@
...
+msgstr "duplikaatti paketti: {}"
+
+#: src/install.rs:975
+msgid "could not find all required packages:"
+msgstr "ei pystytty löytämään kaikkia vaadittuja paketteja"
+
+#: src/install.rs:982
+msgid "\n"
+"    {:missing} (wanted by: {:stack})"
+msgstr ""
```
Comment (Morganamilo) [inline po/fi.po:446]: Not translated.

**Comment by Morganamilo** [comment]
looks good to me at a glance apart from the couple comments.

File: po/fi.po (hunk 1 above)
Comment (keinhen) [reply po/fi.po]: Fixed this in the latest commit

File: po/fi.po [hunk 4]
```diff
@@ -408,9 +408,9 @@ msgstr "ei pystytty lukemaan kansiota: {}"
 
 #: src/install.rs:778
 msgid "{} is a directory\n"
-"{} on kansio\n"
+"\n"
 ""
-msgstr ""
+msgstr "{} on kansio\n"
```
Comment (Morganamilo) [inline po/fi.po:413]: You need to follow the exact formatting as above.

File: po/fi.po (hunk 3 above)
Comment (keinhen) [reply po/fi.po]: Fixed this in the latest commit. 

I gather the {:text} are variables to be replaced by package names? 

In this context I wasn't able to add in a satisfactory, simple translation so I had left it empty, but this time commit I offered in at least a placeholder.

File: po/fi.po (hunk 4 above)
Comment (keinhen) [reply po/fi.po:413]: My bad, sorry. Should be alright in the latest with identical formatting to msgid""

File: po/fi.po (hunk 3 above)
Comment (Morganamilo) [reply po/fi.po]: Yes though I realise it's not working so I need to look into that hmmm

File: po/fi.po (hunk 4 above)
Comment (Morganamilo) [reply po/fi.po:413]: Yes. I only commented on the first occurrence too. You need to do this elsewhere.

Comment (keinhen) [reply po/fi.po:414]: Ah gotcha. I gave it a once over for formatting rigor.

**Comment by Morganamilo** [comment]
Looks good now from what I can see. Though I am getting a little worn out with all these :P Thanks

**Comment by keinhen** [comment]
> Looks good now from what I can see. Though I am getting a little worn out with all these :P Thanks

Great! And no problem, can imagine it can be hard to keep the focus up after a few rounds :D

**Pull request merged**